includes/admin/class-wcmp-export.php

```diff
@@ -41,6 +41,18 @@ class WCMP_Export
     public const DEFAULT_POSITIONS = [2, 4, 1, 3];
     public const SUFFIX_CHECK_REG  = "~^([a-z]{1}\d{1,3}|-\d{1,4}\d{2}\w{1,2}|[a-z]{1}[a-z\s]{0,3})(?:\W|$)~i";
 
+    public const SHIPPING_METHOD_FLAT_RATE        = "flat_rate";
+    public const SHIPPING_METHOD_LEGACY_FLAT_RATE = "legacy_flat_rate";
+    public const SHIPPING_METHOD_TABLE_RATE       = "table_rate";
+    public const SHIPPING_METHOD_LOCAL_PICKUP     = "local_pickup";
+
```

Comment:
moet hier ook niet `betrs_shipping `, om Bolder Elements WooCommerce Table Rate Shipping te ondersteunen?

Reply:
Nee, dat wordt in de js geregeld, maar ook alleen maar omdat de naam niet overeenkomt met het standaard format. 

Er is geen lijstje meer met "allowed shipping methods" want dat gaat stuk als je custom dingen toevoegt. Hij kijkt nu gewoon echt naar wat er is ingevuld in die package types setting en dan alleen of het NIET in de disallowed methods staat.

Reply:
oke nice